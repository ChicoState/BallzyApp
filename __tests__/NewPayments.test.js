'use strict';

import 'react-native';
import React from 'react';
//import React from 'react';
import NewPay from '../src/components/payments/NewPayments';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <NewPay />
  ).toJSON();
  expect(tree).toMatchSnapshot();

});
