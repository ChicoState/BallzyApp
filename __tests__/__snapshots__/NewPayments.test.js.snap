exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "#00BFFF",
      "flex": 1,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 40,
        "fontWeight": "bold",
        "marginBottom": 75,
        "marginTop": 75,
        "textAlign": "center",
      }
    }>
    New Card
  </Text>
  <TextInput
    onChangeText={[Function]}
    placeholder="Name for the Card"
    style={
      Object {
        "backgroundColor": "#F7FFF7",
        "borderColor": "black",
        "borderWidth": 2,
        "height": 40,
        "margin": 12,
      }
    } />
  <TextInput
    onChangeText={[Function]}
    placeholder="Card Number"
    style={
      Object {
        "backgroundColor": "#F7FFF7",
        "borderColor": "black",
        "borderWidth": 2,
        "height": 40,
        "margin": 12,
      }
    } />
  <TextInput
    onChangeText={[Function]}
    placeholder="Expiration Month"
    style={
      Object {
        "backgroundColor": "#F7FFF7",
        "borderColor": "black",
        "borderWidth": 2,
        "height": 40,
        "margin": 12,
      }
    } />
  <TextInput
    onChangeText={[Function]}
    placeholder="Expiration Year"
    style={
      Object {
        "backgroundColor": "#F7FFF7",
        "borderColor": "black",
        "borderWidth": 2,
        "height": 40,
        "margin": 12,
      }
    } />
  <TextInput
    onChangeText={[Function]}
    placeholder="Card CVC"
    style={
      Object {
        "backgroundColor": "#F7FFF7",
        "borderColor": "black",
        "borderWidth": 2,
        "height": 40,
        "margin": 12,
      }
    } />
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "#F7FFF7",
          "borderColor": "black",
          "borderWidth": 1,
          "fontSize": 20,
          "margin": 20,
          "marginLeft": 20,
          "textAlign": "center",
        }
      }>
      Submit
    </Text>
  </View>
</View>
`;
